src/frontend/get-jira-configuration.ts

```diff
@@ -38,22 +35,32 @@ export async function getInstallation(client, subscription) {
 			subscription.repoSyncState?.numberOfSyncedRepos || 0;
 		response.data.jiraHost = subscription.jiraHost;
 
-		response.data.syncStatus === "FAILED" && sendFailedStatusMetrics(id);
+		response.data.syncStatus === "FAILED" && sendFailedStatusMetrics(id, reqLog);
 
 		return response.data;
 	} catch (err) {
+		reqLog.error(
+			{ installationId: id, error: err, uninstalled: err.code === 404 },
+			"Failed connection"
```

Comment:
/nit I would rather say "Failed getting installation" hence current message depends on who is calling it